src/styles/variables/border.js

```diff
@@ -2,6 +2,19 @@ export const borderRadius = {
   fullRadius: '100%',
   partialRadius: '50%',
   edgeRadius: '3%',
+  submitButtonRadius: '0.4rem',
+};
+
+export const buttonRadius = {
+  default: 'none',
```

Comment:
Since these are associated with only the button can you move them into the button component